src/common/bmpbndl.cpp

```diff
@@ -506,6 +506,13 @@ wxIcon wxBitmapBundle::GetIconFor(const wxWindow* window) const
     return GetIcon(GetPreferredBitmapSizeFor(window));
 }
 
+/* static */
+wxBitmap wxBitmapBundle::GetBitmapFor(const wxBitmapBundle& bundle,
+                                      const wxWindow* window)
+{
+    return bundle.GetBitmapFor(window);
+}
+
```

Comment:
Sorry, but there is no point in having this at all if it's implemented like this. I thought your original function did something differently from this, i.e. just silently returned `wxBitmap()` instead of asserting when either the bundle was invalid or the window was null. Looking at the code again, I see that the existing `GetBitmapFor()` already doesn't assert for invalid bundles, only for invalid window. But at least here the window is never invalid, so we just don't need this at all and can just always use `m_bmpXXX.GetBitmapFor(this)` in the code above, can't we?